test/quantization/fx/test_numeric_suite_fx.py

```diff
@@ -460,6 +460,7 @@ def forward(self, x0):
         self.assert_types_for_matched_subgraph_pairs(results, expected_types, mp, mq)
 
     @skipIfNoFBGEMM
+    @unittest.skip("Broken by https://github.com/pytorch/pytorch/pull/62608, need dtype inference support")
```

Comment:
What is the ETA for dtype inference support?

Reply:
@vkuzo will be working on that, I think maybe after he has some time, not sure about the ETAs..